content/en/docs/ops/setup/standalone-operator/index.md

```diff
@@ -1,196 +1,141 @@
 ---
-title: Standalone Operator Quick Start Evaluation Install [Experimental]
-description: Instructions to install Istio in a Kubernetes cluster for evaluation.
+title: Standalone Operator Install [Experimental]
+description: Instructions to install Istio in a Kubernetes cluster using the Istio operator.
 weight: 11
 keywords: [kubernetes, operator]
 aliases:
 ---
 
-This guide installs Istio using the standalone Istio operator. The only dependencies
-required are a supported Kubernetes cluster and the `kubectl` command. This
-installation method lets you quickly evaluate Istio in a Kubernetes cluster on
-any platform using a variety of profiles.
+This guide installs Istio using the standalone Istio
+[operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+The only dependencies required are a supported Kubernetes cluster and the `kubectl` command.
 
-To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+{{< warning >}}
+To install Istio for production use, we recommend [installing with {{< istioctl >}}](/docs/setup/install/istioctl/)
 instead, which is a stable feature.
+{{< /warning >}}
 
 ## Prerequisites
 
 1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
 
 1. Check the [Requirements for Pods and Services](/docs/setup/additional-setup/requirements/).
 
-## Installation steps
-
-1. Install Istio using the operator with the demo profile:
+1. Deploy the operator:
 
     {{< text bash >}}
     $ kubectl apply -f https://preliminary.istio.io/operator.yaml
     {{< /text >}}
 
-{{< warning >}}
-This profile is only for demo usage and should not be used in production.
-{{< /warning >}}
-
-1. (Optionally) change profiles from the demo profile to one of the following profiles:
-
-{{< tabset cookie-name="profile" >}}
-
-{{< tab name="demo" cookie-value="permissive" >}}
-When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
-mutual TLS traffic. Clients send plaintext traffic unless configured for
-[mutual TLS migration](/docs/tasks/security/mtls-migration/). This profile is installed during the first step.
+    This command creates the operator custom resource definition and deploys the controller.
```

Comment:
line 30 borders on too much information imo.  Seems like this could be simplified as either
`This command deploys the Istio operator and a sample profile`

or if you want the detail:

```
This command creates the following resources to deploy the operator:
- The `istio-operator` namespace
- The Istio operator custom resource definition
- The istio operator custom resource which defines how the operator should be deployed
- The `istio-operator-metrics` service to access operator metrics
- The `istio-operator` deployment to start and maintain a running operator
- Necessary Istio operator RBAC rules
```